catalog/resource_sql_table.go

```diff
@@ -64,6 +64,7 @@ type SqlTableInfo struct {
 	ClusterID           string            `json:"cluster_id,omitempty" tf:"computed"`
 	WarehouseID         string            `json:"warehouse_id,omitempty"`
 	Owner               string            `json:"owner,omitempty" tf:"computed"`
+	TableID             string            `json:"table_id,omitempty" tf:"computed"`
```

Comment:
Table ID is an output-only field, so we can remove `omitempty` to ensure TF doesn't allow users to set it.
```suggestion
	TableID             string            `json:"table_id" tf:"computed"`
```

To verify that this won't break users, I created a table with the current version of the provider (so Table ID was not in state), then applied this change, and there is no diff. However, after refresh/apply, the table ID appears.